pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyServiceStarter.java

```diff
@@ -131,30 +138,35 @@ public ProxyServiceStarter(String[] args) throws Exception {
             // load config file
             config = PulsarConfigurationLoader.create(configFile, ProxyConfiguration.class);
 
-            if (!isBlank(zookeeperServers)) {
-                // Use zookeeperServers from command line
-                config.setZookeeperServers(zookeeperServers);
+            if (isBlank(metadataStoreUrl)) {
```

Comment:
Can you add a simple unit test to verify these changes and prevent future regressions?

Reply:
I think The unit test is already covered by `ProxyServiceStarterTest`

Reply:
Well I don't see any retro compatibility test. You modified all the tests to not use `zookeeperServers` anymore and that's ok. But I don't see a test which verify what happens now with zookeeperServers